src/utils/email/emailPlatformMap.ts

```diff
@@ -0,0 +1,10 @@
+import { EmailPlatformConstants } from '@constants/email';
+import { EmailService } from '@services/email.service';
+import { Transporter, SentMessageInfo } from 'nodemailer';
+
+export const transpoterMapper: {
+  [k in EmailPlatformConstants.EmailPlatforms]: () => Transporter<SentMessageInfo>;
```

Comment:
this signature definition looks a bit complex. Can it be encapsulated into its own type?

Reply:
also, there is a typo in "transpoter"

Reply:
Changed!